libs/sdk-model/src/dashboard/filterContext.ts

```diff
@@ -47,6 +47,18 @@ export interface IDashboardAttributeFilterParent {
  */
 export type DashboardAttributeFilterSelectionMode = "single" | "multi";
 
+/**
+ * Item that is used to validate availability of attribute element.
+ *
+ * The item is either metric or is converted to metric (in case of attribute, label, or fact) by backend
```

Comment:
can we comment how it is converted, which aggregation functions are used? Just to describe behavior to reader. Thx

Reply:
Fixed :)